aws/variables.tf

```diff
@@ -72,7 +96,7 @@ variable "modules_info" {
     vpc = {
       create = true
       id = ""
-      private_subnets = []
+      private_subnets_cidr_blocks = [] #["172.16.0.0/21", "172.16.16.0/21", "172.16.32.0/21", "172.16.48.0/21", "172.16.64.0/21"]
```

Comment:
We don't this comment - default or we don't need a comment for it

Reply:
a8b41b2e33894b8e8a0f31c796bee510e62b9291
removed